ote_sdk/ote_sdk/entities/scored_label.py

```diff
@@ -31,11 +31,16 @@ def name(self) -> str:
         return self.label.name
 
     @property
-    def id(self) -> ID:
+    def id_(self) -> ID:
         """
         Returns the label id.
         """
-        return self.label.id
+        return self.label.id_
+
+    @property
+    def id(self) -> ID:
+        """Gets or sets the id of a Model"""
```

Comment:
it does not set, I guess
why Model?

Reply:
setter is required 
or
should it not be DEPRECTATED?

Reply:
Thanks for finding that Ilya, I've corrected it.